projects/chef-infra-server.md

```diff
@@ -0,0 +1,90 @@
+# Chef Infra Server Foundation Team
+
+The Chef Infra Server team is responsible for common libraries and
+applications used to support the mostly erlang based server
+products. This includes:
+  - Chef Server
+  - Push Jobs Server
+  - Push Jobs Client
+
+** Slack Channel **: #chef-server
```

Comment:
You don't want a space between the chars and the *. Otherwise markdown doesn't bold